### DISCUSSION TITLE
Some of my friends keep getting disconnected

### DISCUSSION THREAD

**Discussion by GoldneRozu** [FakeFishGames/Barotrauma#13788] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Some people from my server keep getting disconnected and its not everyone either its only some people.
![image_2024-04-18_215503502](https://github.com/FakeFishGames/Barotrauma/assets/167501659/6c5754bf-f62b-4174-a903-d639179ea188)
That is the Error I get just before they get kicked.  Also the ai bots get that error as well, And their ping also shoots up to over 800-1000. I did turn off ddos protection and raise the limit to 10000

### Reproduction steps

I start my dedicated server, they are able to join while in the menu
I start a new campaign, they are fine while we are docked.
we leave station, and they are fine for a few min and then get kicked and cant rejoin because they keep getting that error from above

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I suspect this could have something to do with some of the mods you're using, could you please post your mod list here? What's the error message when a client gets disconnected?

**Reply by GoldneRozu** [reply]
I will double check, but they all told me they get no errors, it just takes them back to the lobby
![image](https://github.com/FakeFishGames/Barotrauma/assets/167501659/4be668f2-6e16-490f-8259-4170a5a7b0cd)
![image](https://github.com/FakeFishGames/Barotrauma/assets/167501659/03612adc-632d-48ff-ac82-f024cc1c7246)
![image](https://github.com/FakeFishGames/Barotrauma/assets/167501659/154721d3-079c-4819-9712-8d5f895544ee)
![image](https://github.com/FakeFishGames/Barotrauma/assets/167501659/6e19c39d-992a-443b-9d34-c4a06a6459ee)
That is the complete mod list, along with the order

**Reply by GoldneRozu** [reply]
One person is getting this error "unexpectederror not handeld by unlockachivements" this is new to me though, I am going to look into this a little on my own as well
 I have been told that error is coming from Robotrauma, they are also getting an error for clientside can't download prefabs

**Reply by GoldneRozu** [reply]
Okay I might have found the issue, I will let you know, Okay yeah thank you for your time I believe I have fixed my issue

**Reply by greyco1** [reply]
> Okay I might have found the issue, I will let you know, Okay yeah thank you for your time I believe I have fixed my issue

PLEASE, WHAT WAS THE FIX, I NEED THE FIX, HALF OF MY CREW CANNOT JOIN AND FOR THE LIFE OF ME IDK WHAT IT COULD BE
